src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -44,11 +45,32 @@ export const RateGasRow: FC<RateGasRowProps> = memo(
     swapperName,
     swapSource,
     networkFeeFiatUserCurrency,
+    deltaPercentage,
     onClick,
   }) => {
     const translate = useTranslate()
     const { isOpen, onToggle } = useDisclosure()
 
+    const rateContent = useMemo(() => {
+      if (!rate) return null
+      return (
+        <Skeleton isLoaded={!isLoading}>
+          <RawText color='text.subtle' fontWeight='medium'>
+            1 {sellAssetSymbol} = {rate} {buyAssetSymbol}
+            {deltaPercentage && (
+              <RawText
+                as='span'
+                color={deltaPercentage.startsWith('+') ? 'green.500' : 'red.500'}
```

Comment:
preferably-blocking: this should be a pure number (positive/negative) as a string.
Then, can use an expression where:

- 0 is a no-op and returns null (similar to `!rate`)
- gt(0) is green
- lt(0) is red

And prepend with +/-

Reply:
Fixed in [368e0e7](https://github.com/shapeshift/web/pull/8970/commits/368e0e7d5e69c209ef99aa463df33ae023c8efbb) and [995f7d7](https://github.com/shapeshift/web/pull/8970/commits/995f7d7995abc7834decc4936576948c5ba6b3c6)